core/executor/src/native_executor.rs

```diff
@@ -140,18 +74,18 @@ pub struct NativeExecutor<D> {
 	fallback: WasmExecutor,
 	/// Native runtime version info.
 	native_version: NativeVersion,
-	/// The default number of 64KB pages to allocate for Wasm execution.
-	default_heap_pages: Option<u64>,
+	/// The number of 64KB pages to allocate for Wasm execution.
+	initial_heap_pages: Option<u64>,
 }
 
 impl<D: NativeExecutionDispatch> NativeExecutor<D> {
 	/// Create new instance.
-	pub fn new(default_heap_pages: Option<u64>) -> Self {
+	pub fn new(initial_heap_pages: Option<u64>) -> Self {
 		NativeExecutor {
 			_dummy: Default::default(),
 			fallback: WasmExecutor::new(),
 			native_version: D::native_version(),
-			default_heap_pages,
+			initial_heap_pages: initial_heap_pages,
```

Comment:
Do we have any consistent style on whether to use shortened form here or not? I would have just put `initial_heap_pages` and not repeated the line (like the one you removed). It's not in the styleguide so nothing is right or wrong.

Reply:
ah, this is one thing that I forgot to fix actually! I asked @cmichi to come up with something different than `default_heap_pages` since we already have `DEFAULT_HEAP_PAGES` and proposed `initial_heap_pages` but now I think it is even worse